src/UriTemplate.php

```diff
@@ -122,23 +124,25 @@ private static function expandMatch(array $matches, array $variables): string
                 continue;
             }
 
+            /** @var mixed */
```

Comment:
The point of this is to allow static analysers to consider the type as genuinely anything, rather than "don't know, don't show warnings".